src/BlockTypes/OrderConfirmation/Status.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes\OrderConfirmation;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * Status class.
+ */
+class Status extends AbstractOrderConfirmationBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'order-confirmation-status';
+
+	/**
+	 * This renders the content of the block within the wrapper.
+	 *
+	 * @param \WC_Order $order Order object.
+	 * @param string    $permission Permission level for viewing order details.
+	 * @param array     $attributes Block attributes.
+	 * @return string
+	 */
+	protected function render_content( $order, $permission = false, $attributes = [] ) {
+		if ( ! $permission ) {
+			// phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+			return '<p>' . wp_kses_post( apply_filters( 'woocommerce_thankyou_order_received_text', esc_html__( 'Thank you. Your order has been received.', 'woo-gutenberg-products-block' ), null ) ) . '</p>';
```

Comment:
What are your thoughts on formatting this section similar to the next one for readability-reasons:

```suggestion
			return '<p>' . wp_kses_post( 
				apply_filters( 
					'woocommerce_thankyou_order_received_text', 
					esc_html__( 'Thank you. Your order has been received.', 'woo-gutenberg-products-block' ), 
					null 
				)
			 ) . '</p>';
```